.circleci/config.yml

```diff
@@ -48,7 +48,6 @@ jobs:
             TEST_FILES="$(circleci tests glob "spec/**/*_spec.rb" | circleci tests split --split-by=timings)"
 
             bundle exec rspec --format progress \
-                            --format RspecJunitFormatter \
```

Comment:
why did you change this?

Reply:
It was crashing my CircleCI tests (and I don't even know if need JunitFormatter